roles/tpa_single_node/vars/main.yml

```diff
@@ -18,46 +18,48 @@ tpa_single_node_pg_user: guac
 tpa_single_node_pg_user_passwd: guac1234
 tpa_single_node_pg_ssl_mode: disable
 
-# Storage Service (S3 or Minio)
+# Storage Service - S3 or Minio
 tpa_single_node_storage_endpoint: # <Enter minio storage endpoint, For AWS this field is not necessary>
-tpa_single_node_storage_region: # <AWS S3 Storage region> # For Minio just keep us-west-1
+tpa_single_node_storage_region: eu-west-1 # <AWS S3 Storage region> - For Minio just keep us-west-1
 tpa_single_node_storage_access_key: "{{ lookup('env', 'TPA_STORAGE_ACCESS_KEY') }}"
 tpa_single_node_storage_secret_key: "{{ lookup('env', 'TPA_STORAGE_SECRET_KEY') }}"
-tpa_single_node_storage_bombastic_bucket: bombastic-default # <bombastic storage bucket name>
-tpa_single_node_storage_v11y_bucket: v11y-default # <v11y storage bucket name>
+tpa_single_node_storage_bombastic_bucket: bombastic-rhtpa
+tpa_single_node_storage_vexination_bucket: vexination-rhtpa
+tpa_single_node_storage_v11y_bucket: v11y-rhtpa
 
-
-# SQS_SERVICE
-tpa_single_node_event_bus_type: # Either kafka or sqs
+# QUEUE Service - Kafka or SQL
+tpa_single_node_event_bus_type: sqs # Either <kafka> or <sqs>
 tpa_single_node_event_access_key_id: "{{ lookup('env', 'TPA_EVENT_ACCESS_KEY_ID') }}" # Kafka Username or AWS SQS Access Key ID
 tpa_single_node_event_secret_access_key: "{{ lookup('env', 'TPA_EVENT_SECRET_ACCESS_KEY') }}" # Kafka password or AWS SQS Secret Access Key
+tpa_single_node_event_endpoint: "{{ lookup('env', 'TPA_EVENT_ENDPOINT') }}"
 
-### Kafka fields
-# <Only needed for Kafka service, For AWS this field is not necessary>
+### Kafka - Only needed for Kafka service
 tpa_single_node_kafka_bootstrap_servers: "{{ lookup('env', 'TPA_EVENT_BOOTSTRAP_SERVER') | default('tpa_single_node_default_empty', true) }}"
 # Security protocol used for Kafka user authentication.
-tpa_single_node_kafka_security_protocol: # Example - SASL_PLAINTEXT
+tpa_single_node_kafka_security_protocol: SASL_PLAINTEXT
 # Kafka Authentication Mechanism which is spec.kafka.listeners.authentication.type in Block letters.
-tpa_single_node_kafka_auth_mechanism: # Example - SCRAM-SHA-512
+tpa_single_node_kafka_auth_mechanism: SCRAM-SHA-512
```

Comment:
This could be added with  a comment to explaining for which type of event bus is appropriate

Reply:
@desmax74, the variable name is explicit enough. 
Actually I'm not sure we even need a variable here.
What are the other possible values that could be used by rhtpa configuration ?

Reply:
Not by rhtpa configuration, but what the kafka provider ask, the options are:

- GSSAPI (Kerberos) 
- PLAIN, 
- SCRAM SHA-256 
- SCRAM SHA-512
- OAUTHBEARER

In doubt is better if we use this value but commented as a suggestion